fuel-vm/src/checked_transaction.rs

```diff
@@ -841,7 +863,6 @@ mod tests {
     // input
     #[quickcheck]
     fn min_fee_coin_input(
```

Comment:
The purpose of this test to test different `gas_price` as well, could you return it back, please?=)

Reply:
I removed it because it isn't used, since the `gas_price` isn't attached to the tx anymore.

Reply:
But this test is for testing how fee calculation works in general, not how it works within the transaction.